packages/tldraw/src/lib/shapes/video/VideoShapeUtil.tsx

```diff
@@ -151,20 +153,19 @@ export class VideoShapeUtil extends BaseBoxShapeUtil<TLVideoShape> {
 		return <rect width={toDomPrecision(shape.props.w)} height={toDomPrecision(shape.props.h)} />
 	}
 
-	override toSvg(shape: TLVideoShape) {
-		return <image href={serializeVideo(shape.id)} width={shape.props.w} height={shape.props.h} />
+	override async toSvg(shape: TLVideoShape) {
+		const image = await serializeVideo(this.editor, shape)
```

Comment:
we no longer update `props.time` as the video plays. We should really remove the prop entirely. I don't want to pass it in as there's no longer any way to control it

Reply:
it's hard to say but it may be that the `props.time` is temporarily gone? even if we don't bring back sync, we might want to be able to save the time at a certain starting position.
but i don't feel too strongly about this.